src/components/Login.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+function Login() {
+  const [login, setLogin] = React.useState(false);
+  function BtnLog() {
+    setLogin(!login);
+  }
+  return (
+    <NavLink exact to="/Screen">
+      <button type="button" onClick={BtnLog}>Login</button>
```

Comment:
I guess you can just remove the function BtnLog, and write:

          <button type="button" onClick={() => setLogin(!login)}>Login</button>